packages/d3fc-webgl/src/shaders/point/square/baseShader.js

```diff
@@ -0,0 +1,19 @@
+import shaderBuilder, { vertexShaderBase, fragmentShaderBase } from '../../shaderBuilder';
```

Comment:
How come these files are called `baseShader`? What are they the base of?

Reply:
I think they are called `baseShader` as they are the base for each shape that can then be added to. I will update the file name to `shader`